src/components/VoiceRecorder/VoiceRecorder.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import Mic from '@material-ui/icons/Mic';
+import './VoiceRecorder.css';
+
+export default class VoiceRecorder extends React.Component {
+  state = {
+    record: false,
+    mediaRecorder: '',
+    chunks: '',
+    audioURL: '',
+    iconsColor: 'black'
+  };
+
+  render() {
+    const styles = {
+      color: this.state.iconsColor
+    };
+    return (
+      <div className="container-for-recorder">
```

Comment:
We follow B.E.M. class naming convention, which is an acronym for Block, Element, Modifier.

Example:
```
/* Block */
.VoiceRecorder { ... }

/* Element */
.VoiceRecorder__button { ... }

/* Modifier */
VoiceRecorder--recording { ... }
```

By convention our `VoiceRecorder` componet (or block) will have a `.VoiceRecorder` classname. If you need to add state class when it is recording you can do so with `--` like so `.VoiceRecorder--recording`.

Child elements we denote them with `__` so for example `VoiceRecorder__button`.

You should check this [article](https://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/).. many companies use this convention, even Google at some of their projects.

Reply:
Just rename the class name to be B.E.M. like. Don't forget to update your CSS.